CefSharp.Core/Internals/ClientAdapter.h

```diff
@@ -193,6 +194,10 @@ namespace CefSharp
             //CefFindHandler
             virtual DECL void OnFindResult(CefRefPtr<CefBrowser> browser, int identifier, int count, const CefRect& selectionRect, int activeMatchOrdinal, bool finalUpdate);
 
+			//CefAccessibilityHandler
```

Comment:
These should be moved into `RenderClientAdapter`. They're only called as part of `CefRenderHandler`

Reply:
Also spaces not tabs